### PR TITLE
Team creation front-end and some other things

### PR DESCRIPTION
**Pull request by BirGer05**

### CODE REVIEW

File: scrum_planner/resources/views/teams/creating_team.blade.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+@endsection
+
+@section('content')
+
+<div class="container d-flex align-items-center justify-content-center text-center p-5">
+    <div class="text-white">
+
+        <h1 class="mb-3">Create new team</h1>
+
+        <form action={{ url('/sign-in') }} method="POST">
```
Comment (Banyik) [inline scrum_planner/resources/views/teams/creating_team.blade.php]: Why is this here?

**Review by Banyik — CHANGES_REQUESTED** [review]
There is a sign in form without any reason.

**Comment by BirGer05** [comment]
Fixed the issu

File: scrum_planner/resources/views/teams/creating_team.blade.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+            <label class="d-flex justify-content-start align-items-start pe-3 fw-bold form-label" for="teamMembers">
+                Team members
+            </label>
+
+            <select class="selectpicker" data-live-search="true">
+                <option data-tokens="ketchup mustard">Hot Dog, Fries and a Soda</option>
+                <option data-tokens="mustard">Burger, Shake and a Smile</option>
+                <option data-tokens="frosting">Sugar, Spice and all things nice</option>
+            </select>
```
Comment (Banyik) [inline scrum_planner/resources/views/teams/creating_team.blade.php]: What the ... is this?!

**Review by Banyik — CHANGES_REQUESTED** [review]

File: scrum_planner/resources/views/teams/creating_team.blade.php (hunk 2 above)
Comment (BirGer05) [reply scrum_planner/resources/views/teams/creating_team.blade.php]: A simple place holder

Comment (Banyik) [reply scrum_planner/resources/views/teams/creating_team.blade.php]: This is not conventional, nor appropriate.

**Review by Banyik — CHANGES_REQUESTED** [review]
Bad placeholder

**Comment by BirGer05** [comment]
Removed all placeholder

File: scrum_planner/app/Http/Controllers/TeamController.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            return abort(401);
+        }
+
+        $newTeam = Team::create(['team_name' => $fields['team_name'], 'scrum_master' => Auth::user() -> id]);
+        foreach ($fields['members'] as $member) {
+            try {
+                TeamMember::create(['team_id' => $newTeam -> id, 'user_id' => $member]);
+            } catch (\Throwable $th) {
+                //throw $th;
+            }
```
Comment (Banyik) [inline scrum_planner/app/Http/Controllers/TeamController.php]: Missing error message

**Review by Banyik — CHANGES_REQUESTED** [review]
Missing error message on catch

**Comment by BirGer05** [comment]
It now counts how many user could not be added to the team, and shows it to the user creating the team

**Review by Banyik — APPROVED** [review]
finally

**Pull request merged**